doc/api/process.md

```diff
@@ -1684,6 +1684,107 @@ the [`'warning'` event][process_warning] and the
 [`emitWarning()` method][process_emit_warning] for more information about this
 flag's behavior.
 
+## `process.parseArgs([argv[, options]])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `argv` {Array<string>|Object} (Optional) Array of argument strings; defaults
+  to [`process.argv.slice(2)`](process_argv). If an Object, the default is used,
+  and this parameter is considered to be the `options` parameter.
+* `options` {Object} (Optional) The `options` parameter, if present, is an
+  object supporting the following property:
+  * `expectsValue` {Array<string>|string} (Optional) One or more argument
+    strings which _expect a value_ when present
```

Comment:
That's not how we _usually_ write optional parameters ("If an..., it is ...")

https://nodejs.org/docs/latest-v12.x/api/crypto.html#crypto_cipher_update_data_inputencoding_outputencoding

I'm worried the over verbosity might make it more complicated to understand.

Reply:
I've addressed this somewhat, but I find the linked documentation too terse.  🤷